packages/vouching/test/Vouching.test.js

```diff
@@ -24,11 +27,23 @@ contract('Vouching', function ([_, tokenOwner, vouchingOwner, developer, transfe
 
   context('with token', async function () {
     const dependencyName = 'dep';
+    const attributeID = 0;
 
-    beforeEach(async function () {
+    beforeEach('TPL setup', async function () {
+      this.jurisdiction = await BasicJurisdiction.new({ from: jurisdictionOwner });
       this.token = await ZepToken.new({ from: tokenOwner });
+      const initializeData = encodeCall('initialize', ['address', 'uint256'], [this.jurisdiction.address, attributeID]);
+      await this.token.sendTransaction({ data: initializeData, from: tokenOwner });
+      this.validator = await ZEPValidator.new(this.jurisdiction.address, attributeID, { from: validatorOwner });
+      await this.jurisdiction.addValidator(this.validator.address, "ZEP Validator", { from: jurisdictionOwner });
+      await this.jurisdiction.addAttributeType(attributeID, false, false, ZERO_ADDRESS, 0, 0, 0, "can transfer", { from: jurisdictionOwner });
+      await this.jurisdiction.addValidatorApproval(this.validator.address, attributeID, { from: jurisdictionOwner });
+      await this.validator.addOrganization(organization, 100, "ZEP Org", { from: validatorOwner });
+      await this.validator.issueAttribute(tokenOwner, { from: organization });
+      await this.validator.issueAttribute(developer, { from: organization });
       await this.token.transfer(developer, lotsaZEP, { from: tokenOwner });
       this.vouching = await Vouching.new(minStake, this.token.address, { from: vouchingOwner });
+      await this.validator.issueAttribute(this.vouching.address, { from: organization });
```

Comment:
Can we add some tests to ensure TPL is working as expected on this end and maybe for the token too?